packages/appservice-rest/src/Utilities/KuduServiceUtility.ts

```diff
@@ -201,11 +201,10 @@ export class KuduServiceUtility {
     public async deployWebAppImage(appName: string, images: string, isLinux: boolean) {
         try {
             core.debug(`DeployWebAppImage - appName: ${appName}; images: ${images}; isLinux:${isLinux}`);
-            if (!isLinux) {
-                throw new Error("Windows Containerized web app is not available for Publish profile auth scheme.");
-            }
             console.log(`Deploying image ${images} to App Service ${appName}`);
-            let headers = {'LinuxFxVersion': `DOCKER|${images}`};
+
+            let fxVersionName = isLinux ? 'LinuxFxVersion' : 'WindowsFxVersion';
+            let headers = {fxVersionName: `DOCKER|${images}`};

```

Comment:
>images [](http://example.com/codeflow?start=52&length=6)

Add check to confirm this is not empty ?

Reply:
Done.

Reply:
This might not check if images is string.empty.